ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install_utils.sh

```diff
@@ -42,7 +42,6 @@ apt-get -y install numactl \
                    libnl-route-3-dev \
                    net-tools \
                    libsecret-1-0 \
-		   ansible \
```

Comment:
should we have this change to the LTS image also?

Reply:
Let me check. I am not sure if we released Moneo for LTS

Reply:
So the LTS version doesn't have Moneo. Pretty sure we decided not to include it.

Reply:
Do we need ansible in the LTS version? We recently pulled all these apt installs into the ubuntu/common/install_utils.sh file, but if we need to specialize by image version again we can do that...